docs/reference/contrib/forms.rst

```diff
@@ -37,17 +37,17 @@ Within the ``models.py`` of one of your apps, create a model that extends ``wagt
         intro = RichTextField(blank=True)
         thank_you_text = RichTextField(blank=True)
 
-    FormPage.content_panels = [
-        FieldPanel('title', classname="full title"),
-        FieldPanel('intro', classname="full"),
-        InlinePanel('form_fields', label="Form fields"),
-        FieldPanel('thank_you_text', classname="full"),
-        MultiFieldPanel([
-            FieldPanel('to_address', classname="full"),
-            FieldPanel('from_address', classname="full"),
-            FieldPanel('subject', classname="full"),
-        ], "Email")
-    ]
+        content_panels = AbstractEmailForm.content_panels + [
+            FieldPanel('title', classname="full title"),
```

Comment:
If we're appending to `AbstractEmailForm.content_panels`, presumably we don't need to add the title field here?